app/src/main/java/nerd/tuxmobil/fahrplan/congress/favorites/StarredListAdapter.kt

```diff
@@ -26,15 +26,9 @@ class StarredListAdapter internal constructor(
 
 ) {
 
-    private val nowMoment = Moment.now()
-
     @ColorInt
     private val pastSessionTextColor = ContextCompat.getColor(context, R.color.favorites_past_session_text)
 
-    override fun initViewSetup() {
-        nowMoment.setToNow()
-    }
-
```

Comment:
:thinking: I remember doing this here: a73a506fc054a07951036b6b5674f32293f80624. It is not documented in the Git history but I remember there was some reason to initialize the moment as early as possible. Maybe this triggers something with you.

Reply:
Well, now there's no good reason left. The `Moment` was set to "now" in `SessionAdapter.getView()` shortly before `setItemContent()` was called. And the body of `setItemContent()` is the only place that accesses the extension property `Session.tookPlace` which is the only place that uses the `Moment`. So every time `setItemContent()` was called the moment was reset to the current time. With this change the behavior stays the same (give or take a couple of milliseconds).

Finding out whether or not your original method of doing this was really necessary is left as an exercise for code historians.

Reply:
Looks like it can be simplified. :heavy_check_mark: Successfully tested on Pixel 2, Android 10 with system date and time set back to 36C3 day 1.

_As a code historian I am always interested if the former me has left behind helpful commit messages. Never give up hoping._